install/sessions.csv

```diff
@@ -540,7 +541,8 @@ TKAR,GeoNet,GNS,TRIMBLE NETR9,GPS+GLO,30s,0,geonet,trimble_netr9 x4,2014-07-23T2
 TKHL,GeoNet,GNS,TRIMBLE NETR9,GPS+GLO,30s,0,geonet,trimble_netr9 x4,2014-06-18T00:00:20Z,9999-01-01T00:00:00Z
 TORY,GeoNet,GNS,TRIMBLE NETRS,GPS,30s,4,geonet,trimble_5700 x4,2005-04-20T05:00:00Z,2006-12-30T00:00:00Z
 TORY,GeoNet,GNS,TRIMBLE NETRS,GPS,30s,4,geonet,trimble_5700 x4,2007-01-03T21:00:00Z,2011-11-16T21:16:30Z
-TORY,GeoNet,GNS,TRIMBLE NETRS,GPS,30s,4,geonet,trimble_5700 x4,2011-11-16T23:11:00Z,9999-01-01T00:00:00Z
+TORY,GeoNet,GNS,TRIMBLE NETRS,GPS,30s,4,geonet,trimble_5700 x4,2011-11-16T23:11:00Z,2017-07-17T23:51:58Z
+TORY,GeoNet,GNS,TRIMBLE NETR9,GPS+GLO,30s,0,geonet,trimble_netr9 x4,2017-07-17T23:51:59Z,9999-01-01T00:00:00Z
```

Comment:
referring to my comment for L193-194. Here is excpected to have a new session entry, because TORY had a NetRS to NetR9 swap. But for NetR9-NetR9 swaps I'm not sure if we need to have a new line in the session.csv file.